Sources/OktaOidc/Common/Internal/OIDAuthorizationService+Okta.swift

```diff
@@ -35,7 +35,7 @@ extension OKTAuthorizationService {
 
             delegate?.didReceive(response)
             guard let response = response as? HTTPURLResponse else {
-                callback(nil, error ?? OktaOidcError.api(message: "Authentication Error: No response", underlyingError: error))
+                callback(nil, error ?? OktaOidcError.api(message: "Authentication Error: No response", underlyingError: nil))
```

Comment:
Why is the underlying error nil here?

Reply:
Because we have the optional `nil` coalescing operator here. 
`error ?? OktaOidcError.api(message: "Authentication Error: No response", underlyingError: error))`

if we don't have `error` then it means it's `nil` here.

Reply:
Or should we return `api` error though?

Reply:
I think an API error, with underlying error, makes more sense here. That way we have a consistent list of error types a developer can read from, and if there is an error below the covers, they can access the raw URL response.

Reply:
Agree. And fixed :)